stream-helpers/signals.ts

```diff
@@ -0,0 +1,117 @@
+import {
```

Comment:
Is this used?

Reply:
valve uses the array signal and faucet uses boolean signal. I'm thinking about exporting faucet as part of the api (it's in test-helpers export)